js/models/order/orderFulfillment/CryptoDelivery.js

```diff
@@ -8,6 +8,12 @@ export default class extends BaseModel {
     };
   }
 
+  get max() {
```

Comment:
I used to use the name `max` as well and I found `constraints` works better because it would support something like this:

```
  get constraints() {
    return {
      maxTransactionIDLength: 512,
      minTransactionIDLength: 25,
    };
  }
```

Reply:
Sounds reasonable. Fixed.